app/actions/views/channel.js

```diff
@@ -333,6 +353,12 @@ export function refreshChannel(channelId) {
     };
 }
 
+export function refreshChannelWithRetry(channelId) {
```

Comment:
Same comment as above with `loadPostsIfNecessary` vs `loadPostsIfNecessaryWithRetry`. I don't think we need to keep both if we always want to retry